### ISSUE TITLE
[QUESTION] How do I connect to Babelfish using JTDS? 

### ISSUE THREAD

**Issue by markedcahinde** [babelfish-for-postgresql/babelfish_extensions#593]
We have provisioned an AWS Aurora Cluster (PostgreSQL 14.3) with Babelfish for Aurora PostgreSQL 2.1  enabled.
It seems though that we're having trouble connecting using JTDS driver encountering the following error log

```
Caused by: java.sql.SQLException: I/O Error: DB server closed connection.
	at net.sourceforge.jtds.jdbc.TdsCore.nextToken(TdsCore.java:2481)
	at net.sourceforge.jtds.jdbc.TdsCore.login(TdsCore.java:632)
	at net.sourceforge.jtds.jdbc.JtdsConnection.<init>(JtdsConnection.java:371)
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:184)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.lambda$0(JDBCDataSource.java:191)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.lambda$1(JDBCDataSource.java:210)
	at org.jkiss.dbeaver.utils.SecurityManagerUtils.wrapDriverActions(SecurityManagerUtils.java:94)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:208)
	... 15 more
Caused by: java.io.IOException: DB server closed connection.
	at net.sourceforge.jtds.jdbc.SharedSocket.readPacket(SharedSocket.java:852)
	at net.sourceforge.jtds.jdbc.SharedSocket.getNetPacket(SharedSocket.java:731)
	at net.sourceforge.jtds.jdbc.ResponseStream.getPacket(ResponseStream.java:477)
	at net.sourceforge.jtds.jdbc.ResponseStream.read(ResponseStream.java:114)
	at net.sourceforge.jtds.jdbc.TdsCore.nextToken(TdsCore.java:2368)
```

We are using the latest JTDS driver (1.3.1) and this connection string
`jdbc:jtds:sqlserver://<db-url>:<db-port>/<db-name>`

Has someone also experienced the same?

**Comment by kuntalghosh** [comment]
JTDS is not a supported driver (it's not in the list of drivers in the docs at https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/babelfish-connect.html)
 
Could you please provide a simplified code sample that connects to Babelfish endpoint and then throws this error?
Also, if you're using SSL, could you please turn off SSL and retry once?

**Comment by markedcahinde** [comment]
I'm using DBeaver to connect to the cluster. I've used the jTDS connection driver for SQL Server with version 1.3.1 of the driver, so I've got no code. Is there a direct way to disable SSL in Babelfish besides Aurora cluster parameter groups?

**Comment by suprio-amzn** [comment]
@markedcahinde: Is there a particular reason you want to use the jTDS driver? Dbeaver can be configured to use other JDBC drivers -- could you try the Microsoft SQL Server's JDBC driver? 

https://github.com/dbeaver/dbeaver/wiki/Database-drivers

Also, please use Babelfish via TDS (beta) as the database:

<img width="778" alt="DBeaver databases" src="https://user-images.githubusercontent.com/93294502/191044245-0cdb5eb1-25f8-48b8-80c7-941bdbe177c8.png">

**Comment by markedcahinde** [comment]
Right, so this connection via jTDS on DBeaver is only a test. We do have a legacy application using jTDS driver to connect to SQL Server 2014, and now in progress to migrate that to Babelfish. For a number of reasons, moving away from jTDS is not quite the option, so we're looking for a right way to do this. We have tested DBeaver to try and identify any configurations to make it work, then eventually try to apply that against the legacy.

Will read up more and try Babelfish via TDS (beta) in DBeaver. But hopefully someone has tried using jTDS with Babelfish and provide feedback.

**Comment by suprio-amzn** [comment]
@markedcahinde : As mentioned by @kuntalghosh above https://github.com/babelfish-for-postgresql/babelfish_extensions/issues/593#issuecomment-1247856660, jTDS is not one of the supported drivers as yet.

**Comment by markedcahinde** [comment]
Thanks for the response @suprio-amzn! I believe jTDS' latest version (1.3.1 released circa 2013) supports TDS 7.1,  with that, is jTDS and other legacy drivers part of Babelfish roadmap to support in future? 

I've tried to use Babelfish via TDS driver in DBeaver, while it has successfully connected to the database, the DBO schema wasn't listed; but I believe this is different from the jTDS query at hand.
